src/App.jsx

```diff
@@ -14,45 +16,99 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_METHOD_ALPHABETICALLY = 'alphabetically';
+const SORT_METHOD_BY_LENGTH = 'length';
+
+const getViewGoods = (goods, sortMethod, isReversed) => {
+  let sortedGoods = [...goods];
+
+  switch (sortMethod) {
+    case SORT_METHOD_ALPHABETICALLY:
+      sortedGoods = sortedGoods.sort((a, b) => a.localeCompare(b));
+      break;
+
+    case SORT_METHOD_BY_LENGTH:
+      sortedGoods = sortedGoods.sort((a, b) => a.length - b.length);
+      break;
+
+    default:
+      break;
+  }
+
+  if (isReversed) {
+    sortedGoods = sortedGoods.reverse();
```

Comment:
same with reverse() method, it will mutate sortedGoods, so you can just write this here:
 `sortedGoods.reverse();`